src/Compilers/Test/Resources/Core/CompilerTestResources.csproj

```diff
@@ -391,70 +392,15 @@
     <EmbeddedResource Include="NetFX\ValueTuple\TupleElementNamesAttribute.cs" />
     <Content Include="SymbolsTests\CustomModifiers\GenericMethodWithModifiers.cpp" />
     <None Include="SymbolsTests\Metadata\public-and-private.cs" />
+    <Compile Include="ResourceLoader.cs" />
```

Comment:
:bulb: In my experience, grouping `<Compile>` items with items of other types breaks the ability of the project to insert new `<Compile>` items at the correct location (proper group and alphabetical order). Unless this has changed, I would prefer to see these items remain in their own `<ItemGroup>`.